skfp/fingerprints/avalon.py

```diff
@@ -12,7 +12,64 @@
 
 
 class AvalonFingerprint(FingerprintTransformer):
-    """Avalon fingerprint."""
+    """
+    Avalon fingerprint.
+
+    The implementation uses RDKit [1]_. This is a hashed fingerprint, where
+    fragments are computed based on atom environments. Avalon is a path-based
+    fingerprint.
+
+    The fingerprint is based on the Avalon toolkit [2]_ described in [3]_.
+
+    Parameters
+    ----------
+
+    fp_size : int, default=512
+        Size of output vectors, i.e. number of bits for each fingerprint. Must be
+        positive.
+
+    count : bool, default=False
+        Whether to use binary or count fingerprints.
+
+    sparse : bool, default=False
+        Whether to return sparse matrix.
+
+    n_jobs : int, default=None
+        Number of parallel jobs. If -1, then the number of jobs is set to the number of CPU cores.
+
+    verbose : int, default=0
+        Verbosity level.
+
+    Attributes
+    ----------
+    n_features_out : int
+        Number of output features.
+
+    References
+    ----------
+    .. [1] RDKit implementation of Avalon fingerprints
+        https://www.rdkit.org/docs/source/rdkit.Avalon.pyAvalonTools.html
+    .. [2] Avalon toolkit
+        https://sourceforge.net/projects/avalontoolkit/
+    .. [3] Gedeck, P., Rohde, B., & Bartels, C. (2006).
+        "QSAR − How Good Is It in Practice? Comparison of Descriptor Sets on an Unbiased
+        Cross Section of Corporate Data Sets."
+        Journal of Chemical Information and Modeling, 46(5), 1924–1936. doi:10.1021/ci050413p
```

Comment:
Use reference format like in Atom Pairs. We should stick to one formatting. It will also create nice hyperlinks. For authors, use MLA format instead of APA (first one on Google Scholar).
```
    .. [3] `Gedeck, Peter, Bernhard Rohde, and Christian Bartels
        "QSAR − How Good Is It in Practice? Comparison of Descriptor Sets on an Unbiased
        Cross Section of Corporate Data Sets."
        J. Chem. Inf. Model. 2006, 46, 5, 1924–1936`_
```